src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/debian/Microsoft.NETCore.Runtime.CoreCLR.pkgproj

```diff
@@ -23,7 +23,8 @@
     <ArchitectureSpecificNativeFile Include="$(BinDir)mscorlib.ni.dll"/>
     <ArchitectureSpecificLibFile Include="$(BinDir)mscorlib.dll"/>
     <ArchitectureSpecificToolFile Include="$(BinDir)crossgen"/>
-
+    <ArchitectureSpecificToolFile Include="$(BinDir)libclrjit.so"/>
```

Comment:
> Unpublished standalone with dotnet run needs mscorlib.ni packaging to run out of the additional probe path

In the current PR, clrjit.dll is next to coreclr.dll, just like mscorlib.ni.dll. Thus, there should not be any concern picking it up.

When JIT moves to its own package, it will be listed in the app.deps.json for the unpublished standalone app (like the CoreCLR nuget package) and whatever additional probing path the host uses to lookup CoreCLR.dll from its package location, the host will use similar logic to lookup the JIT and pass its path to the runtime for initialization.